docs/docs/docs/getting-started/installation.md

```
@@ -295,6 +295,18 @@ devcontainer build --workspace-folder .
 devcontainer up --workspace-folder .
 ```
 
+#### Import Sample Data
+
+1. Once the server is running, open a new terminal session.
+2. Open a bash session inside the running container:
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)